lte/gateway/c/core/oai/tasks/nas5g/src/M5GIdentityRequest.cpp

```diff
@@ -25,54 +25,43 @@ IdentityRequestMsg::IdentityRequestMsg(){};
 IdentityRequestMsg::~IdentityRequestMsg(){};
 
 // Decode IdentityRequest Message and its IEs
-int IdentityRequestMsg::DecodeIdentityRequestMsg(
-    IdentityRequestMsg* identity_request, uint8_t* buffer, uint32_t len) {
+int IdentityRequestMsg::DecodeIdentityRequestMsg(uint8_t* buffer,
+                                                 uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.